packages/ui/src/App.tsx

```diff
@@ -91,8 +92,10 @@ const App = () => {
               <Route path="/forgot-password/reset" element={<ResetPassword />} />
               <Route path="/forgot-password/:method" element={<ForgotPassword />} />
 
-              {/* social sign-in pages */}
+              {/* continue */}
```

Comment:
```suggestion
              {/* Continue */}
```
better to elaborate the meaning of continue?